packages/select/src/components/query-list/queryList.tsx

```diff
@@ -229,10 +227,10 @@ export class QueryList<T> extends React.Component<IQueryListProps<T>, IQueryList
         return undefined;
     }
 
-    private getActiveIndex() {
+    private getActiveIndex(items = this.state.filteredItems) {
```

Comment:
nit: since this is private it doesn't matter much, but if you call `getActiveIndex()` elsewhere, maybe it'd be better just to use an explicit non-optional argument.

Reply:
🤷‍♂️